extra/markdown-extra/MarkdownRuntime.php

```diff
@@ -22,11 +22,41 @@ public function __construct(MarkdownInterface $converter)
 
     public function convert(string $body): string
     {
-        // remove indentation
-        if ($white = substr($body, 0, strspn($body, " \t\r\n\0\x0B"))) {
-            $body = preg_replace("{^$white}m", '', $body);
-        }
+        $body = $this->commonWhitespace($body);
```

Comment:
I don't think we should change the content, except for maybe the beginning of the lines. Here, you are removing all occurrences.